recipes/zlib-ng/all/test_package/enough.c

```diff
@@ -0,0 +1,574 @@
+/* enough.c -- determine the maximum size of inflate's Huffman code tables over
```

Comment:
Please take a look at our docs and try to shrink this as much as possible 🙏 

https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#minimalistic-source-code